### PR TITLE
[1.19] Point to k8s release-1.19 branch for tests

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:
And update the tutorial doc to say release-1.19 instead
of release-1.18.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: contrib/test/integration/main.yml [hunk 1]
```diff
@@ -25,15 +25,15 @@
...
-        k8s_git_version: "release-1.18"
+        k8s_git_version: "release-1.19"
         k8s_github_fork: "kubernetes"
         crio_socket: "/var/run/crio/crio.sock"
       when: "(cgroupv2 is undefined) or (cgroupv2 == False) | bool"
 
     - name: clone build and install kubernetes for cgroup v2
       include: "build/kubernetes.yml"
       vars:
-        k8s_git_version: "master"
```
Comment (saschagrunert) [inline contrib/test/integration/main.yml:36]: What is the reason that we do not stick to the master here?

Comment (umohnani8) [reply contrib/test/integration/main.yml:36]: Yeah, I wasn't entirely sure about this one. In 1.18 we were using @giuseppe's branch. I was testing out if this works with the release-1.19 branch, and looks like it does. But if we want to keep cgroupv2 pointing to master, I will revert it here.

Comment (saschagrunert) [reply contrib/test/integration/main.yml:36]: Right now they're still the same with some slight delay, but if we apply Code Thaw (in some weeks) in Kubernetes, then we will not be able to test the latest master changes any more. I think we could simply stick to master, WDYT?

Comment (umohnani8) [reply contrib/test/integration/main.yml]: Sure, fixed!

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM once tests are happy

**Comment by haircommander** [comment]
/hold

we've decided to wait until we stop forwarding from master to do this (should be in a week or two)

**Comment by umohnani8** [comment]
/hold cancel

This can be merged now. @mrunalp @haircommander PTAL

**Pull request merged**